lib/oauth2.js

```diff
@@ -7,27 +7,21 @@ function Strategy(options, verify) {
   options = options || {};
   options.authorizationURL = options.authorizationURL || 'https://www.linkedin.com/oauth/v2/authorization';
   options.tokenURL = options.tokenURL || 'https://www.linkedin.com/oauth/v2/accessToken';
-  options.scope = options.scope || ['r_basicprofile'];
-  options.profileFields = options.profileFields || null;
+  this.scope = options.scope = options.scope || ['r_basicprofile'];
```

Comment:
We should document why we need this duplication. Other than that the workaround sounds legit ;-)